app/src/components/NetworkSettingsCard/__tests__/DisableDiscoveryCache.test.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+import * as React from 'react'
+import { Provider } from 'react-redux'
+import { mount } from 'enzyme'
+import noop from 'lodash/noop'
+
+import * as AllConfig from '../../../config'
+import { LabeledToggle } from '@opentrons/components'
+import { DisableDiscoveryCache } from '../DisableDiscoveryCache'
+
+import type { State } from '../../../types'
+import type { Config } from '../../../config/types'
+
+jest.mock('../../../config/selectors')
+
+const MOCK_STATE: State = ({ mockState: true }: any)
+
+const getConfig: JestMockFn<[State], Config> = AllConfig.getConfig
```

Comment:
The typechecker is complaining on this file because the mock configuration return values you're giving it aren't full-on configs. Obviously having all of your mock values be full configs would be really annoying in the context of this text because most of the fields have nothing to do with what we're testing.

Luckily, we can tell Flow to allow us to pass partial Configs as our mock return values:

```suggestion
const getConfig: JestMockFn<[State], $Shape<Config>> = AllConfig.getConfig
```

`$Shape<Obj>` tells Flow "treat this thing like it's a full `Obj`, even if its missing fields, as long as the fields that _are_ there are the correct type"